src/sass/pages/_signUp.scss

```diff
@@ -0,0 +1,94 @@
+.signUp {
+  display: flex;
+
+  @include respond(phone) {
+    flex-direction: column;
+  }
+
+  &__left,
+  &__right {
+    width: 50%;
+    @include respond(phone) {
+      width: 100%;
+    }
+  }
```

Comment:
isn't this better to handle just using flex, if your container occupy 100% of the screen and you assign a class for the container element to just use flex, you could just change flex-direction for mobile/desktop from column to row avoiding the use of the 50% width

Reply:
I guess the result is the same... I don't know if it's a better practice though. :P

Reply:
@FlaviaCohen for browsers width operations are more expensive, and comes even more explicit when you try to animate or transition things with absolute width values like %